src/VS.Web.CG.Mvc/Templates/RazorPageGenerator/DeletePageModel.cshtml

```diff
@@ -53,7 +53,7 @@ namespace @Model.NamespaceName
                 return NotFound();
             }
 
-            @Model.ViewDataTypeShortName = await _context.@(Model.ModelTypeName)@(inlineIncludes).SingleOrDefaultAsync(m => m.@primaryKeyName == id);
+            @Model.ViewDataTypeShortName = await _context.@(Model.ModelTypeName)@(inlineIncludes).FindAsync(id);
```

Comment:
Not sure what `@(inlineIncludes)` expands too. If it is calls to `Include()` then this would result in generating invalid code because `FindAsync()` is only defined on `DbSet<T>` but not on the type returned by `Include()`. For these cases in which you cannot use `FindAsync()` I would prefer `FirstOrDefaultAsync()` rather than `SingleOrDefaultAsync()` because it leads to simpler SQL.